.github/pull_request_template.md

```diff
@@ -5,20 +5,29 @@
 3. Make sure documentation is updated for your PR!
 
 -->
+# Description
+<!-- Briefly describe the motivation for the change. Please include illustrations where appropriate. -->
 
-**What this PR does / why we need it**:
-<!-- Explain here the context and why you're making the change. What is the problem you're trying to solve. --->
+# Modifications
+<!-- Summarize the key code changes. -->
 
-**Which issue(s) this PR fixes**:
-<!--
-*Automatically closes linked issue when PR is merged.
-Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+## Tests
```

Comment:
can be '#' or is this intended to be under modification? since this is the only child in the whole template

# Modification 
`file.go` -- changed ....
`fileb.go` -- changed ....
`filec.go` -- changed ....

## Tests
- [x] Deploying new and existing standard models
- [ ] Deploying PyFunc models

Reply:
I intended for it to be under Modification but `#` works too.